cfgov/jinja2/v1/offices/_single.html

```diff
@@ -147,17 +147,20 @@ <h3 class="h2 qa-resource-title">
         {{ office.content | safe }}
     </section>
     {% endif %}
-    <aside>
+    <aside class="block
```

Comment:
Instead adding all these classes, would it make sense to combine them into one class? The value would be that you aren't just repeating the same block of classes.